patchwork/app.py

```diff
@@ -219,7 +276,9 @@ def cli(
             # treat --key=value as a key-value pair
             inputs[key] = value
 
-    with logger.panel(f"Patchflow {patchflow} logs") as _:
+    if debug: 
+        if inputs.get("debug") is not None:
```

Comment:
if the debug flag is set the debug value is always true if Im not wrong.

Reply:
fixed.